_docs/Qabel-Protocol-Box.md

```diff
@@ -274,6 +275,7 @@ External:
 
 ```
 {
+is_folder: BOOL, // indicates if external is a folder or a file
```

Comment:
Could you add the type definition of `BOOL` in [this file](https://github.com/Qabel/qabel.github.io/blob/master/_docs/Qabel-Client-Local-Data.md), please?